spock-core/src/main/java/org/spockframework/runtime/extension/builtin/PendingFeatureInterceptor.java

```diff
@@ -12,7 +12,7 @@
   public void intercept(IMethodInvocation invocation) throws Throwable {
     try {
       invocation.proceed();
-    } catch (AssertionError e) {
+    } catch (Throwable t) {
```

Comment:
Catching `Throwable` is a bad idea, you risk swallowing Errors that indicate that the JVM failed, e.g., `OutOfMemoryError`.